src/PKSim.Presentation/Services/ProjectTask.cs

```diff
@@ -231,7 +231,7 @@ public void LoadProjectFromSnapshot()
 
          async Task ProjectFromSnapshot()
```

Comment:
This works great here. However the HeavyWorkManager does not support async stuff propertly. We will have to update that as well